src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final int FIRST_INDEX = 0;
+    private static final int SECOND_INDEX = 1;
```

Comment:
index of what? :) you naming should make your code more readable, it's an opportunity to make it clear for those who will read your code what is going on here. Call them OPERATION_INDEX or sth like that